src/fixto.js

```diff
@@ -326,7 +327,7 @@ var fixto = (function ($, window, document) {
                     }
                 }
             }
-            return top;
+            return top + this.options.top;
```

Comment:
this line is violating the description of the method: "Returns the total outerHeight of the elements passed to mind option. Will return 0 if none."

We will need to change the description if we need to have this.